tutorial/datastructures.po

```diff
@@ -122,12 +123,12 @@ msgid ""
 "Sort the items of the list in place (the arguments can be used for sort "
 "customization, see :func:`sorted` for their explanation)."
 msgstr ""
-"Classe les éléments sur place (les arguments peuvent personnaliser le "
-"classement, voir :func:`sorted` pour leur explication)."
+"Tri les éléments (les arguments peuvent personnaliser le tri, voir :func:"
+"`sorted` pour leur explication)."
 
 #: ../Doc/tutorial/datastructures.rst:91
 msgid "Reverse the elements of the list in place."
-msgstr "Inverse l'ordre des éléments de la liste, sur place."
+msgstr "Inverse l'ordre des éléments de la liste."
```

Comment:
La traduction de `in place` par "sur place" n'est peut être pas idéale, mais l'information est tout de même importante. Si tu as un meilleur traduciton je prend. L'enlever je ne suis pas pour. Pareil qu'avent, le contrate entre `reversed` et `.reverse`.